paasta_tools/utils.py

```diff
@@ -1657,3 +1655,20 @@ def mean(iterable):
     Returns the average value of an iterable
     """
     return sum(iterable) / len(iterable)
+
+
+class NoTtyError(Exception):
+    pass
+
+
+def prompt_pick_one(sequence):
+    if not sys.stdin.isatty():
+        raise NoTtyError
```

Comment:
I understand the need to raise, but sometimes from a user perspective, a stacktrace isn't the right thing to see.

If there is nothing that expects to ever actually catch this, I would prefer it just return None when there is no tty to choose and then let calling code deal with None just like someone quit.